float64 distance_travelled
float64 distance_to_target
